src/containers/OrbitalViewerContainer.jsx

```diff
@@ -74,6 +77,27 @@ class OrbitalViewerContainer extends React.PureComponent {
     }
   }
 
+  getSetOrbitData = (response, randomSource) => {
+    const { data } = response;
+    const { answers, updateAnswer } = this.props;
+    const { pageId } = this.global;
+
+    if (randomSource) {
```

Comment:
if `randomSource` is set, generate a unique key for the orbit viewer on this page. 

One limitation here is that the keys are by pageId so if there are multiple orbit viewers on a page only one random orbit can be persisted. 

The only alternative I can think of is to start putting an ID in the widget and using that as the lookup. Thoughts on if this is worth the time to add? Maybe add an ID and use the page key as a backup?